profiling/proto/service/service.go

```diff
@@ -0,0 +1,3 @@
+package service
```

Comment:
What does this file exist for?

Reply:
So that `make proto` from the project root directory generates the `profiling/proto/service/service.pb.go` file. This is more standardised than expecting a dev to run `protoc` manually, which requires them to generate it in a specific way to keep references to the `.proto` unchanged.

This makes changing the `.proto` and `.pb.go` as easy as running a `make proto`. The underlying `go generate` command in the `Makefile` requires the `//go:generate` comment.